### PR TITLE
PR for #41, #43 and #44

### PR DESCRIPTION
**Pull request by 22PoojaGaur**
PR for 
#41 updated the submodule

#44  It appears we had to include the file names in setup.py for the files to be copied to site-packages. I have tested with pip install . and it is working

#43 updated the flag

@uosl Please take a look

### CODE REVIEW

**Comment by heralden** [comment]
Your approach to solve #44 seems to work, but I see that the docker-intermine-gradle dir gets added to site-packages, outside the intermine_boot dir. I don't think this is good practice, as it could conflict with a different pypi package that happened to have the same name (unlikely but still worth considering).

Could you try following the recommendation at the bottom here: https://python-packaging.readthedocs.io/en/latest/non-code-files.html
According to that, we should place the docker-intermine-gradle submodule inside the `intermine_boot` directory, as it's being depended on by the code.

File: intermine_boot/intermine_docker.py [hunk 1]
```diff
@@ -243,7 +243,9 @@ def create_intermine_builder_container(client, image, env, options):
         'MINE_REPO_URL': os.environ.get('MINE_REPO_URL', ''),
         'MEM_OPTS': os.environ.get('MEM_OPTS', '-Xmx2g -Xms1g'),
         'IM_DATA_DIR': os.environ.get('IM_DATA_DIR', ''),
-        'FORCE_MINE_BUILD': 'true' if options['datapath_im'] else 'false'
+        'FORCE_MINE_BUILD': 'true' if (
+               options['datapath_im'] and not _is_conf_same(env['data_dir', options])
```
Comment (heralden) [inline intermine_boot/intermine_docker.py]: It seems the `]` is misplaced here, should be `_is_conf_same(env['data_dir'], options)`.

**Comment by 22PoojaGaur** [comment]
@uosl handled both comments.

**Comment by heralden** [comment]
Thanks!

Sorry about my pickiness, but there's one more thing we should fix before merging. Now we're including docker-intermine-gradle with seemingly two ways: `MANIFEST.in` and `package_data`. This can be confusing, and we only need one way. MANIFEST.in seems the more recommended way, so can you update the paths in that (simply adding `intermine_boot/` to each path in the file, like you did elsewhere)? And then delete `package_data` from `setup.py`?

**Comment by 22PoojaGaur** [comment]
> Thanks!
> 
> Sorry about my pickiness, but there's one more thing we should fix before merging. Now we're including docker-intermine-gradle with seemingly two ways: `MANIFEST.in` and `package_data`. This can be confusing, and we only need one way. MANIFEST.in seems the more recommended way, so can you update the paths in that (simply adding `intermine_boot/` to each path in the file, like you did elsewhere)? And then delete `package_data` from `setup.py`?

done!

**Review by heralden — APPROVED** [review]
Thank you, I appreciate it! :tada:

**Pull request merged**